cmd/datamon/cmd/flags.go

```diff
@@ -290,11 +295,30 @@ func addTargetFlag(cmd *cobra.Command) string {
 	return c
 }
 
+func addCacheSizeFlag(cmd *cobra.Command) string {
+	c := "cache-size"
+	datamonFlags.bundle.CacheSize = flagext.ByteSize(50 * units.MB)
+	cmd.Flags().Var(&datamonFlags.bundle.CacheSize, c, "The desired size of the memory cache used when streaming is enabled")
```

Comment:
could we update the doc strings to explicitly include a few common available unit strings (e.g. KB, MB, GB)?

opinion here is that `flagext` makes flags more usable only to the extent that users know how to user `flagext`.

Reply:
sure. Doing this.